examples/task_spesific_configuration.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+#
+# To make this run, you probably want to edit /etc/luigi/client.cfg and add something like:
+# DO NOT FORGET to set keep-alive as True.
+#
+
+from time import sleep
+from uuid import uuid4
+
+import luigi
+
+
+class MyTaskHasConfig(luigi.WrapperTask):
+    """
+        Wrapper class for some error and success tasks. Worker won't be shutdown unless there is
+        pending tasks or failed tasks which will be retried. While keep-alive is active, workers
+        are not shutdown while there is/are some pending task(s). This task also has a task level
+        config which is ``upstream-status-when-all``;This configuration is about how workers behave
+        to set ``MyTaskHasConfig`` status according to it's upstream tasks which are ``ErrorTask1``,
+        ``ErrorTask2`` and ``SuccessTask``. If this config is set as ``False``, Any of upstream task
+        ``FAILED`` or ``DISABLED`` will make ``MyTaskHasConfig`` ``FAILED`` or ``DISABLED`` which also means,
+        shut the worker down immediately. While this is ``True``, all upstream tasks should be ``FAILED``
+        or ``DISABLED``, worker to set ``MyTaskHasConfig`` status as ``FAILED`` or ``DISABLED``. ``MyTaskHasConfig`` will
+        remain as PENDING unless all of its upstream task severities (so does not contains SUCCESS tasks)
+        are ``FAILED`` or ``DISABLED``
```

Comment:
Sorry, I was a bit unclear. I imagined that you make the comment like in: https://github.com/spotify/luigi/blob/5bd9b436990d48acc8def675382c29f8fb0e729e/examples/execution_summary_example.py#L16-L46.

However, your text would be reworded to be written to describe the whole example, instead of talking from the perspective of the "MyTaskHasConfig" class.